### ISSUE TITLE
Thank you dev for this wonderful app

### ISSUE THREAD

**Issue by shockergit** [Lambada10/SongSync#123]
Thanks this app is wonderfull. Can you add one more feature to sync the spotify tags likealbum art, title, album, composet, year and artist name of the music also with the lyric. Actually i download music from youtube and it doesn't come with any tags only filenames. It would have been nice if all the spotify tags will get synced with the lyrics

**Comment by BobbyESP** [comment]
Maybe it's a bit selfish of me to say this, but you can do all of that with my [Metadator](https://github.com/BobbyESP/Metadator) app. I'll soon be releasing an update.

**Comment by shockergit** [comment]
> Maybe it's a bit selfish of me to say this, but you can do all of that with my [Metadator](https://github.com/BobbyESP/Metadator) app. I'll soon be releasing an update.

Thanks for the suggestion let me try it

**Comment by shockergit** [comment]
> Maybe it's a bit selfish of me to say this, but you can do all of that with my [Metadator](https://github.com/BobbyESP/Metadator) app. I'll soon be releasing an update.

Good app but for some reasons its not saving album art for some files. Some works fine. My files are all in opus.

**Comment by shockergit** [comment]
> Maybe it's a bit selfish of me to say this, but you can do all of that with my [Metadator](https://github.com/BobbyESP/Metadator) app. I'll soon be releasing an update.

K now im tagging the metadata from Metadator and album art from automatag

**Comment by BobbyESP** [comment]
> > Maybe it's a bit selfish of me to say this, but you can do all of that with my [Metadator](https://github.com/BobbyESP/Metadator) app. I'll soon be releasing an update.
> 
> Good app but for some reasons its not saving album art for some files. Some works fine. My files are all in opus.

Yeah known issue, so please, open an issue in my repo and I'll fix it as soon as possible

**Comment by shockergit** [comment]
> > > Maybe it's a bit selfish of me to say this, but you can do all of that with my [Metadator](https://github.com/BobbyESP/Metadator) app. I'll soon be releasing an update.
> > 
> > 
> > Good app but for some reasons its not saving album art for some files. Some works fine. My files are all in opus.
> 
> Yeah known issue, so please, open an issue in my repo and I'll fix it as soon as possible

Yes thanks the app is awesome

**Comment by jo-chemla** [comment]
@shockergit Could you update the issue title to something more meaningful like `Make SongSync edit song metadata from provider like title, artist, album etc`
